byterun/floats.c

```diff
@@ -26,11 +26,13 @@
 #include "caml/reverse.h"
 #include "caml/stacks.h"
 
-#ifdef _MSC_VER
-#include <float.h>
-#define isnan _isnan
-#define isfinite _finite
-#endif
+#if defined (_MSC_VER)
+#  include <float.h>
+#  if(MSC_VER < 1900)
+#    define isnan _isnan
+#    define isfinite _finite
+#  endif/*(MSC_VER <= 1900)*/
```

Comment:
On Wed, Nov 18, 2015 at 8:03 AM, Mark Shinwell <notifications@github.com>
wrote:

> Should this line say "<" rather than "<="?


​Should say '<' I believe. Sorry, it appears the comment is a copy-paste
error.

-- 
Shayne Fletcher